CODE/VulnScan/v3/_train.py

```
@@ -0,0 +1,391 @@
+    """
+    Saves the trained model and plots the accuracy and loss over epochs.
+
+    Args:
+        model (nn.Module): The trained PyTorch model.
+        save_model_path (str): The path to save the model.
+        accuracy_list (list[float]): List of accuracy values over epochs.
+        loss_list (list[float]): List of loss values over epochs.
+        epochs (int): The number of epochs.
+        model_name (str): The name of the model.
+    """
+    logger.info(f"Saving {model_name} model")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)